File: Day 6/python/sum_of_multiple_of_3_or_5.py
# Modify the previous program such that only multiples of three or five are considered in the sum, e.g. 3, 5, 6, 9, 10, 12, 15 for n=17.
def sum_of_multiple_3_or_5(n):
    sum=0
    for i in range(1,n+1):
        if (i%3==0 or i%5==0):
            sum = sum + i
    return sum
n = int(input('Last digit of sum:'))
print(sum_of_multiple_3_or_5(n))

File: Day 6/javascript/sum_of_multiple_of_3_or_5.js
// # Modify the previous program such that only multiples of three or five are considered in the sum, e.g. 3, 5, 6, 9, 10, 12, 15 for n=17.

number = 17;
let sum = 0;
for (let i = 1; i <= number; i++) {
    if(i%3==0 || i%5==0)
        sum += i;
}
console.log('The sum of natural numbers:', sum);   